static/styles/zulip.css

```diff
@@ -1386,7 +1386,8 @@ td.pointer {
     .messagebox,
     .date_row {
         background-color: hsla(192, 19%, 75%, 0.2);
-        box-shadow: inset 2px 0 0 0 hsl(0, 0%, 27%), -1px 0 0 0 hsl(0, 0%, 27%);
+        box-shadow: inset 3px 0 0 -1px hsl(0, 0%, 27%),
```

Comment:
Let's put a comment here explaining why we're doing the negative spread radius, for posterity.